great_ape_safe/ape_api/ebtc.py

```diff
@@ -1604,3 +1606,86 @@ def cdp_withdraw_debt(self, cdp_id, debt_withdrawable_amount):
 
         # 2.2. icr decreased
         assert new_icr < prev_icr
+
+    def adjust_cdp_with_collateral(
```

Comment:
i'd like to get some test data from here. basically tenderly sims of what this function generates for hints and compare against the sdk. if you set the current cdp values. then come up with input values we can compare and confirm it uses correct data and hints

Reply:
That's a great idea, will create a ticket to optimize hint estimation as you suggest. Would you mind if we resolve outside of this PR? For the purpose of the operation where it is being used today, hint optimization isn't that meaningful imo.

Reply:
Operation will use the following parameters:
```
cdp ID: 0xd0a7a8b98957b9cd3cfb9c0425abe44551158e9e0129d4c80000000000000001
Coll increase amount: 1.6005948082974533e+20, Coll decrease amount: 0
Debt increase amount: 3.920599286570441e+18
Coll Amount hint: 245945434665773231950, Borrow Amount hint: 6489902503698886000
Upper hint: 0xea9375bde335a7322467a33b204df974148ee7dd0129edb1000000000000000f, Lower hint:
0x1fc3410361ce2bd67d60f4ccce00e2d310cc3f6b0129d6480000000000000002
```

And the following is the resulting tx, with a gas of `551633`:
```
Transaction sent: 0x619e00fd3f94ea223c5e8a022696bfffe4b90ad9cf61d0c02258889498bec620
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 2
  IBorrowerOperations.adjustCdpWithColl confirmed   Block: 19583140   Gas used: 551633 (4.60%)
```